
FIXME a few more to take care of
FIXME merge include files that are included only once into the .c file

git grep HOSTS_C

---
 drivers/scsi/mvme147.c |    3 ---
 1 file changed, 3 deletions(-)

--- a/drivers/scsi/mvme147.c
+++ b/drivers/scsi/mvme147.c
@@ -125,9 +125,6 @@ static int mvme147_bus_reset(struct scsi
 	return SUCCESS;
 }
 
-#define HOSTS_C
-
-#include "mvme147.h"
 
 static struct scsi_host_template driver_template = {
 	.proc_name		= "MVME147",
